spyderlib/plugins/shortcuts.py

```diff
@@ -807,6 +816,22 @@ def apply_settings(self, options):
         self.table.save_shortcuts()
         self.main.apply_shortcuts()
 
+    def load_button_clicked(self, change):
+        selected = self.combo_keybindings.currentText().lower()
+        reset_shortcuts()

```

Comment:
Do we really need to do this? What would happen with user defined shortcuts in this case?